### PR TITLE
SI-Portfolio-pt-2018

### PR DESCRIPTION
**Pull request by joshuawilliam6439**

### CODE REVIEW

**Comment by candre717** [comment]
LGTM

**Comment by fredynavas** [comment]
LGTM

File: joshua2:5.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<head>
+  <title>Joshua Nungesser's First Website</title>
+    <link rel="stylesheet" type="text/css" href="css/joshua.css" />
+<style>
```
Comment (chdastolfo) [inline joshua2:5.html]: This works for here, but ideally, you would move these styles into joshua.css

File: css/christina.css [hunk 2]
```diff
@@ -1,51 +0,0 @@
-body{
```
Comment (chdastolfo) [inline css/christina.css]: WHERE HAS IT GONE. I MISS MY CSS. PUT IT BACK PLEASE.

File: css/joshua2:5.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    width: 150%;
+}
+
+.intro-shelf {
+    margin-top: 0;
+    color: rgb(0, 191, 255);
+    background-image: linear-gradient(180deg, rgba(255,255,255,0) 60%, #fff),linear-gradient(70deg, #dbedff 32%, #ebfff0);
+    border-bottom: #fff;
+}
+github-5eb8e2e3716d.css:6
```
Comment (chdastolfo) [inline css/joshua2:5.css]: I think you can probably safely delete all of these references to github here.

File: css/joshua2:5.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+}
+frameworks-521cbf980c80.css:14
+* {
+    box-sizing: border-box;
+}
+user agent stylesheet
+div {
+    display: block;
+}
+Inherited from body.logged-in.env-production.emoji-size-boost.page-dashboard
```
Comment (chdastolfo) [inline css/joshua2:5.css]: Same here - you can delete anything that isn't a css style attribute.

File: css/joshua2:5.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
+body{
+    background-image: url('../img/SI Project Echo Lake.jpg');
+    color: #fff;
```
Comment (chdastolfo) [inline css/joshua2:5.css]: Why do you have 2 css files? All of this can probably go in the same file.

**Comment by rferati** [comment]
👍  👍  👍

**Pull request closed without merging**